cmd/summary.go

```diff
@@ -36,12 +37,33 @@ func init() {
 }
 
 var summaryCmd = &cobra.Command{
-	Use:   "summary",
-	Short: "Genarate a summary of the vpa recommendations in a namespace.",
-	Long:  `Gather all the vpa data in a namespace and generaate a summary of the recommendations.`,
+	Use:   "summary [namespace]",
+	Short: "Generate a summary of the vpa recommendations in a namespace (default: all namespaces).",
+	Long: `Gather all the vpa data in a namespace and generate a summary of the recommendations.
+By default the summary will be about all VPAs in all namespaces.`,
+	Args: cobra.ArbitraryArgs,
```

Comment:
Curious why you chose to use args instead of a namespace flag here, which would be consistent with other Kube-adjacent CLI tools.

Reply:
TBH I didn't even think of using a flag here. I was thinking more in terms of how I might want to use the summary command and thought `goldilocks summary <namespace>` made sense to me, but I now think that `goldilocks summary --namespace <namespace>` also makes just as much sense.

Which would you prefer?

Reply:
I generally lean towards a flag myself.

Reply:
👍